### ISSUE TITLE
Can I change pushes to other RTMP sources in real time without interruption? 

### ISSUE THREAD

**Issue by Ennorath** [ossrs/srs#1343]
For example:
I have 2 users in real time.
1st user's OBS -> srs -> rtmp://twitch.com/user1
2nd user's live from mobile -> srs -> rtmp://twitch.com/user2

And new user want to start live now with his OBS Live 
3rd user OBS -> srs -> rtmp://twitch.com/user3

To add him, I need change config file and restart all srs, which will interrupt 1st and 2nd user's lives on 30-60 seconds. But can i do it without restarting srs?

Can i change config file without interrupting server's process.

**Comment by hakimzulkufli** [comment]
You can simply change the stream URL, such as `rtmp://mystreamserver.com/live/<stream_name>`.

However, that raises another question. How do we secure the server with authentication tokens so that not anyone can simply start their own stream?

**Comment by Ennorath** [comment]
No, i mean it will be not only on my site, i want to send this stream to other sites
